src/app/client/src/app/modules/learn/components/course-consumption/course-player/course-player.component.ts

```diff
@@ -94,6 +94,7 @@ export class CoursePlayerComponent implements OnInit, OnDestroy {
   isConnected = false;
   dropdownContent = true;
   showForceSync = true;
+  primaryCategoryType:any;
```

Comment:
@sohailamjad12   rename variable name to isPrimaryCategoryPIAA (boolean variable) as you are saving boolean value.